### PR TITLE
FLUME-2929 Update License file for 1.7 release

### PR DESCRIPTION
**Pull request by simonati**
add xz,lz4,kafka-client
remove zookeeper
fixed name jackson-mapper-acl,jackson-core-acl

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -527,3 +527,46 @@ For
    CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR
    TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF
    THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+For
+  xz-<version>.jar:
+
+    http://git.tukaani.org/?p=xz-java.git;a=blob_plain;f=COPYING
```
Comment (mpercy) [inline LICENSE]: No need to specify the URL here; it may even change in the future since this doesn't point to a git hash or anything

**Review by bessbd — APPROVED** [review]
LGTM, +1

**Review by mpercy — APPROVED** [review]
+1

Looks good, thanks for fixing this!

**Comment by bessbd** [comment]
I'm about to commit this

**Pull request closed without merging**